assets/quill-cursors.scss

```diff
@@ -91,26 +97,22 @@ $transition-func: cubic-bezier(0.250, 0.460, 0.450, 0.940);
     }
   }
 
-  .ql-cursor-flag:hover,
-  .ql-cursor-flag.show-flag,
-  .ql-cursor-caret-container:hover + .ql-cursor-flag {
-    opacity: 1;
-    visibility: visible;
-    transition: none;
-  }
-
   .ql-cursor-flag.no-delay[style] {
     transition-delay: unset !important;
   }
 
   .ql-cursor-caret-container {
-    margin-left: -$spacing-sm;
-    padding: 0 $spacing-sm;
```

Comment:
Why did we remove this? It means we need to exactly overlap the cursor to show the flag, right? Which might be a bit fiddly?